Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/SubgraphInterface.h

```diff
@@ -173,10 +173,17 @@ namespace ScriptCanvas
 
             const Outs* GetOuts(const AZStd::string& in) const;           
 
+            // meaningless (and empty) if not an object
+            const AZStd::string& GetParentClassName() const;
+
             bool IsActiveDefaultObject() const;
 
+            bool IsBaseClass() const;
+
             AZ::Outcome<bool> IsBranch(const AZStd::string& in) const;
 
+            bool IsClass() const;
```

Comment:
Can you add more comment for these public functions? like what case is not a class?

and I am little confused about user nodeable, nodeable, function definition, function call, etc. I think function is also a nodeable? but instead of c++ implemented internal state, function is like graph implemented?